src/sinks/util/encoding/encoding_config.rs

```diff
@@ -0,0 +1,142 @@
+use crate::sinks::util::encoding::{
```

Comment:
Our pattern has been to avoid the repetition of module names in filenames. I recommend `encoding/with_default.rs` and something like `encoding/without_default.rs` or `encoding/no_default.rs`.